modules/meshrenderinggl/src/datastructures/rasterization.cpp

```diff
@@ -31,7 +31,7 @@
 
 namespace inviwo {
 
-Document Rasterization::getDescription() const {
+Document Rasterization::getInfo() const {
     Document doc;
     doc.append("p", "Rasterization functor.");
     return doc;
```

Comment:
This should describe what kind of rasterisation i.e. fancy/line/sphere etc. could also append the getInfo of the mesh data?

Reply:
was meant for the implementation one, think this one can be abstract

Reply:
That's the general rasterization info, so only visible if an inherited class does not override the document. Could make it purely virtual of course, but I think the fallback is fine in case of a very simple rasterizer processor.